src/components/data-management/SamplesTable.jsx

```diff
@@ -48,6 +50,18 @@ const exampleDatasets = [
   },
 ];
 
+// If a sample exists in projects but not in the samples file,
+// there is an error uploading the sample. We should show an error for that sample.
+const sampleUploadErrorTemplate = {
```

Comment:
Also I would call it `sampleCreateErrorTemplate` to be more specific.

Reply:
I think this will only be used in this component, nowhere else, so I don't think putting it inside initial state is proper. Besides that,the values inside `files` will later on have to be generated dynamically depending on the technology chosen for the file upload once we support other techonlogies (e.g. BD Rhapsody)

Reply:
Resolved the variable is deleted